modules/nf-scil/tracking/localtracking/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: "tracking_localtracking"
+channels:
+  - Docker
+  - Apptainer
+dependencies:
+  - "scilpy"
```

Comment:
Minor comment, but I think everywhere else, `scilpy` is written with a capital S.